lib/nylas/model/attributable.rb

```diff
@@ -21,6 +25,10 @@ def to_h
         attributes.to_h
       end
 
+      private def attribute_definitions
+        self.class.attribute_definitions
```

Comment:
This is a little bit scary to me... I think I'd rather see the call to `class` where it's used rather than having to look in an instance method.

Reply:
+1, don't think this is necessary either

Reply:
Sounds reasonable. Gonna push this up.